packages/react-app/src/constants.js

```diff
@@ -9,8 +9,10 @@ export const BLOCKNATIVE_DAPPID = "0b58206a-f3c0-4701-a62f-73c7243e8c77";
 
 export const ALCHEMY_KEY = "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 
-// RPC_POLL_TIME as 5 seconds to slower the rpc calls at start
-export const RPC_POLL_TIME = 5000;
+// LOCAL_RPC_POLL_TIME as 5 seconds to slower the rpc calls at start
+export const LOCAL_RPC_POLL_TIME = 5000;
+
+export const MAINNET_RPC_POLL_TIME = 30000;
```

Comment:
Can we add / change comments here? Let's explain a little bit better what this does.

We could also mention something about the pollingInterval thing in `useStaticJsonRPC.js`.